Bio/SeqUtils/NucUtils.py

```diff
@@ -0,0 +1,191 @@
+# Copyright 2018 Aleksandra Jarmolinska.  All rights reserved.
+# This file is part of the Biopython distribution and governed by your
+# choice of the "Biopython License Agreement" or the "BSD 3-Clause License".
+# Please see the LICENSE file that should have been included as part of this
+# package.
+"""Functions for dealing with GC/AT content."""
+
+
+from __future__ import print_function
+
+__GC_CONTENT_VALUE = {
+    "G": 1.,
+    "C": 1.,
+    "A": 0.,
+    "T": 0.,
+    "S": 1.,
+    "W": 0.,
+    "N": 0.5
+}
+__GC_CONTENT_VALUE_AMBG = {
+    "M": 0.5,
+    "K": 0.5,
+    "R": 0.5,
+    "Y": 0.5,
+
+    "B": 0.66,
+    "D": 0.33,
+    "H": 0.33,
+    "V": 0.66,
+}
+
+
+def calc_gc_content(seq, interpret_all=False):
+    """Calculate G+C content, returns a fraction (float between 0 and 1).
+
+    Copes mixed case sequences, and with the ambiguous nucleotides S (G or C)
+    and N ( A or C or G or T ) when counting the G and C content.
```

Comment:
There meaning for S and N is isn't clear to me as written.

Reply:
I've expanded this a bit